docs/PUBLISHING.md

```diff
@@ -95,6 +95,8 @@ Run the following command:
 ./scripts/publish/aws-s3.sh <file>
 ```
 
+Also add links to each AWS S3 file in GitHub Releases.
```

Comment:
Might be worth making `GitHub Releases` into a link to https://github.com/resin-io/etcher/releases , and maybe also adding a link to https://github.com/resin-io/etcher/releases/tag/v1.0.0-beta.17 as an example of what to do?